specs/swagger.yml

```diff
@@ -1029,9 +1272,9 @@ definitions:
       description:
         type: string
       createAt:
-        type: number
```

Comment:
This should be `number` as well as `updatedAt` and `microserviceMessageCounts`. Type casting must be done in ioFog agent.